rs/ethereum/ledger-suite-orchestrator/BUILD.bazel

```diff
@@ -161,6 +161,9 @@ rust_ic_test(
         "LEDGER_ARCHIVE_NODE_CANISTER_WASM_PATH": "$(rootpath //rs/ledger_suite/icrc1/archive:archive_canister_u256.wasm.gz)",
     },
     proc_macro_deps = [],
+    tags = [
+        "on_push_to_master",  # since it takes longer than 5 minutes.
```

Comment:
let's give this a name that's more related to the kind of test this is (i.e. "slow test") as opposed to where/when it's run. Otherwise whenever we change where/when it's run we need to update the tags throughout the codebase; also with `on_push_to_master` it's unclear what's special about this test

Reply:
Yeah I considered `slow_test` as well but initially discarded it because we also have tests that are even too slow for the on-push-to-master workflow. So then it might be confusing on when to tag a test with `slow_test`. With `on_push_to_master` a reader directly can see when this test will actually run and won't have to study our CI logic any further.

Reply:
But OTOH I also don't care that much about the name so I'm fine with `slow_test` as well. Do you think we should rename the workflow and job as well?

Reply:
Done.